docs/guides/06-aggregations.md

```diff
@@ -0,0 +1,408 @@
+```js
+function displayGlobalValues() {
+    const v = citiesViz.variables;
+    console.log(`
+        Maximum: ${v.g_max.value}
+        Average: ${v.g_avg.value.toFixed(0)}
+        95th percentile: ${v.g_p95.value}
+    `);
+}
+citiesLayer.on('loaded', displayGlobalValues);
+```
+> Creating a separated function allows easily deactivating the layer later on with `citiesLayer.off`.
+
+Global functions can be easily combined with other capabilities, such as `filter`. For example, you can display just the biggest cities in the world with this little addition to your `viz`:
+```js
+const citiesViz = new carto.Viz(`
+    color: grey
+    width: 10
+    @g_max: globalMax($pop_max)
+    @g_avg: globalAvg($pop_max)
+    @g_p95: globalPercentile($pop_max, 95)
+    filter: ($pop_max > @g_p95)
+`);
+```
+> How about using filtering to display just the biggest city in the world? How would you create a filter for that? Give it a quick try and then keep on working in the guide...
+
+The result using previous global functions should look like this:
+<div class="example-map">
+    <iframe
+        id="guides-aggregations-step-1"
+        src="/developers/carto-vl/examples/maps/guides/aggregations/step-1.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+To open the map at this step, use this [link to the map](/developers/carto-vl/examples/maps/guides/aggregations/step-1.html), where you can check the console messages.
+
+
+### Viewport aggregations
+Sometimes you want to extract relevant information from the current view of the map (the **viewport**), understanding *What am I exactly seeing now?*. And a set of functions is included in CARTO VL to help you in this case.
+
+In an analogous way with the global ones, you have available these functions:
+- `viewportMin`
+- `viewportMax`
+- `viewportSum`
+- `viewportCount`
+- `viewportAvg`
+- `viewportPercentile`
+
+> Obviously, the features that don't pass any possible filter you have placed (hence not visible), will be excluded from the calculations.
+
+You'll practice now to use some of them, adding several to the current viz. First, add some viewport functions to sum the population in the current view and to get the biggest and lowest population values, with this code:
+```js
+const citiesViz = new carto.Viz(`
+    color: grey
+    width: 10
+    @g_max: globalMax($pop_max)
+    @g_avg: globalAvg($pop_max)
+    @g_p95: globalPercentile($pop_max, 95)
+    filter: ($pop_max > @g_p95)
+    // filter: $pop_max == @g_max // biggest city is Tokyo!
+    @v_sum: viewportSum($pop_max)
+    @v_max: viewportMax($pop_max)
+    @v_min: viewportMin($pop_max)
+`);
+```
+> Notice how you can also use a convention to name your "viewport-related" variables, for example with a `v_` prefix.
+
+As a check, show the new variable values in the console
+```js
+function displayViewportValues() {
+    const v = citiesViz.variables;
```

Comment:
Same as above (const v --> const variables). Actually, we can avoid to assign this to a new value. Maybe we should keep calling the viz object `viz` instead of `citiesViz` and then just use `viz.variables.g_max` to be consistent with the rest of the guides.

Reply:
Yeah, that's true. In other guides like '02-Sources' I used different vizs, but here we only have one, so maybe we should be more homogeneous

Reply:
I'm gonna change it to just 'viz'